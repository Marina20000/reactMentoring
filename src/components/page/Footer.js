import React from 'react';
import Inscription from './../captions/Inscription';
import style from './style.css';

export default class Footer extends React.Component {

    render() {
        return <div style={{ height: 100, backgroundColor: 'black', bottom: 0 }}>
            <Inscription className={ style.netflix } inscription='netflixroulette'/>
       </div>;
    }

}