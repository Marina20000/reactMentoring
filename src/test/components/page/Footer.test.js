import React from "react";
import renderer from 'react-test-renderer';
import Footer from './../../../components/page/Footer';

describe('Footer', () => {
    it("testing with snapshot", ()=>{
        const tree = renderer.create(<Footer/>).toJSON();
      expect(tree).toMatchSnapshot();
    })
})