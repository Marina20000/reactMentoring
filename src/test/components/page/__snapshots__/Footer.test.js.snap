// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer testing with snapshot 1`] = `
<div
  style={
    Object {
      "backgroundColor": "black",
      "bottom": 0,
      "height": 100,
    }
  }
>
  <div
    className="netflix"
  >
    netflixroulette
  </div>
</div>
`;
